packages/dev/docs/src/Layout.js

```diff
@@ -254,7 +254,7 @@ export function Layout({scripts, styles, pages, currentPage, publicUrl, children
 
   return (
     <Page scripts={scripts} styles={styles} publicUrl={publicUrl} currentPage={currentPage}>
-      <div className={docStyles.pageHeader} id="header" />
+      <div className={docStyles.pageHeader} id="header" role="banner" />
```

Comment:
Should we use a `<header>` element for this?

Reply:
We could. I’m not sure if we already use header elsewhere. I seem to recall that there can be only one per page.